File: sdk/src/main/java/com/robotemi/sdk/listeners/OnUserInteractionChangedListener.java
package com.robotemi.sdk.listeners;

public interface OnUserInteractionChangedListener {

    /**
     * Listen for user interaction with temi.
     *
     * @param isInteracting true if a user is interacting with temi, otherwise false.
     */
    void onUserInteraction(boolean isInteracting);
}

File: sdk/src/main/java/com/robotemi/sdk/listeners/OnUserInteractionChangedListener.kt
package com.robotemi.sdk.listeners;

interface OnUserInteractionChangedListener {

    /**
     * Listen for user interaction with temi.
     *
     * @param isInteracting true if a user is interacting with temi, otherwise false.
     */
    fun onUserInteraction(isInteracting: Boolean)
}
